web/soubory/blackarrow/program/program.less

```diff
@@ -73,3 +73,258 @@
+}
+
+.program_legenda_inner {
+  align-self: flex-end;
+  flex      : 0 1 auto;
+}
+
+
+.program_legenda_typ.organizator {
+  text-transform: capitalize;
+}
+
+@media (max-width: 800px) {
+  .program_legenda {
+    padding     : 10px 0 3px 8px;
+    margin-right: 20px;
+    margin-left : 20px;
+    margin-top  : -24px;
+  }
+
+  .informaceSpustime {
+    padding    : 4px 4px 0;
+    line-height: 12px;
+  }
+
+  .program_legenda_typ {
+    flex         : 1 1 25%;
+    line-height  : 21px;
+    padding      : 0px 14px 1px 4px;
+    margin-bottom: 0;
+  }
+
+  .program_legenda_typ::before {
+    font-size   : 20px;
+    margin-top  : -3px;
+    margin-right: 10px
+  }
+}
+
+/* oprava zobrazeni odhlasit na stejnem radku */
+.program td>div>form {
+  display      : inline-block !important;
+  padding-right: 3px;
+}
+
+.program td:first-of-type div.program_nazevLinie {}
+
+.program a.programNahled_odkaz[href*="/legendy"]~.program_osobniTyp::before {
+  content     : "Legendy";
+  display     : inline-block;
+  margin-right: 200px;
+}
+
+.program a.programNahled_odkaz[href*="/drd"]~.program_osobniTyp::before {
+  content     : "mDrD";
+  display     : inline-block;
+  margin-right: 200px;
+}
+
+.program a.programNahled_odkaz[href*="/bonusy"]~.program_osobniTyp::before {
+  content     : "Akční hry";
+  display     : inline-block;
+  margin-right: 200px;
+}
+
+.program a.programNahled_odkaz[href*="/epic"]~.program_osobniTyp::before {
+  content     : "Epiky";
+  display     : inline-block;
+  margin-right: 200px;
+}
+
+.program td div.prihlasen,
+.program td div.nahradnik,
+.program td div.organizator {
+  position: relative;
+}
+
+.program .nahradnik form a:before {
+  content  : "odsledovat";
+  display  : inline-block;
+  font-size: 12px;
+}
+
+.program .nahradnik form a {
+  display    : inline-block;
+  white-space: nowrap;
+  width      : 0;
+  height     : 12px;
+  font-size  : 0;
+}
+
+.program .program_osobniTyp {
+  position   : absolute;
+  right      : 0;
+  top        : 0;
+  height     : 100%;
+  width      : 20px;
+  border-left: none;
+  white-space: nowrap;
+  text-indent: 20px;
+  margin     : 0;
+  color      : rgba(255, 255, 255, 0);
+  overflow   : hidden;
+}
+
+.program .programNahled_odkaz~form~b {
+  padding-left: 8px;
+}
+
+.program td .programNahled_odkaz {
+  display      : block;
+  width        : 100%;
+  text-overflow: ellipsis;
+  overflow     : hidden;
+  white-space  : nowrap;
+  min-width    : 85px;
+}
+
+.program .program_osobniTyp::before {
+  position        : absolute;
+  color           : rgba(255, 255, 255, 0.75);
+  transform       : rotate(90deg);
+  transform-origin: left top;
+  left            : 100%;
+  top             : 0;
+  margin-top      : 2px;
+  text-indent     : 0;
+}
+
+.program .programNahled_odkaz~form~form {
+  padding   : 2px 5px 4px;
+  box-shadow: 6px 7px 7px 0 rgba(0, 0, 0, 0.15);
+  position  : absolute !important;
+  top       : 100%;
+  left      : 0;
+}
+
+
+.blahshdawhdhawh {
```

Comment:
Co je tohle?:D

Reply:
to se tam muselo odněkud vykopírovat. Každopádně by se asi mělo taky kouknout na to stylování ať je to přehlednější